test/integration/extensions/filters/network/echo/echo_integration_test.cc

```diff
@@ -59,6 +60,8 @@ TEST_P(EchoIntegrationTest, Hello) {
   EXPECT_EQ("hello", response);
 }
 
+// TODO(mattklein123): This test does not belong here in an extension integration test. Move it
```

Comment:
I'm not following your meaning here... where does it belong?

Reply:
I just mean that this is a test covering core functionality (listener warming) and as such it should be in a "core" integration test. I will make the comment better.

Reply:
Wait, I thought we were keeping all the integration tests together and I think over time there's going to be more where http filters play with network filters or network filters interact.   Can you update me on the planned structure of these tests?

Reply:
I thought this was the plan. ;) 

(This is why I started with such a simple example).

I'm totally fine in not moving integration tests and keeping them where they are, and allowing extensions to be used in "core" integration tests. Thoughts?

Reply:
Can we at least leave them for now and maybe revisit when we have moved more filters, and can see if it makes sense?